.github/workflows/build_package.yml

```diff
@@ -41,4 +41,5 @@ jobs:
         shell: bash
         working-directory: ${{ vars.RUNNER_TEMP }}
         run: python -c "import dsp"
-        
+      - name: Run tests
```

Comment:
Normally, this part will be added as a separated workflow in: https://github.com/stanfordnlp/dspy/pull/283

Reply:
Tests are failing since you would need to install all required deps etc...

Reply:
Ah, I didn't think to poke around to see if another PR was already looking to bring up tests. I'll pull all the testing related changes out and wait for 283 to land to re-add the test.